_Tests/AudibleApi.Tests/L1/AuthorizeTests.cs

```diff
@@ -29,7 +29,16 @@ namespace Authorize_L1
 	[TestClass]
 	public class AuthorizeTests
 	{
-		[TestMethod]
+		/// <summary>
```

Comment:
Thanks for the added explanation. Plz re-read for typos and the like

Reply:
Holy crap. That's... worse than usual lol